dev-infrastructure/modules/cluster-service.bicep

```diff
@@ -8,7 +8,7 @@ param clusterServiceManagedIdentityName string
 param clusterServiceManagedIdentityPrincipalId string
 
 @description('The name of the database to create for CS')
-param csDatabaseName string = 'cluster-service'
+param csDatabaseName string = 'clusters-service'
```

Comment:
Unusure about this rename, but trusting that it works for CS

Reply:
the name is configurable in CS and Mark preferred it like this because it is closer to other setups